packager/packager.cc

```diff
@@ -1259,6 +1283,19 @@ struct PackageSpecBodyWalk {
     }
 
 private:
+    string buildValidLayersStr(const core::GlobalState &gs) {
+        auto &validLayers = gs.packageDB().layers();
+        ENFORCE(validLayers.size() > 0);
+        if (validLayers.size() == 1) {
+            return string(validLayers.front().shortName(gs));
+        }
+        string result = "";
+        for (int i = 0; i < validLayers.size() - 1; i++) {
+            result += fmt::format("{}, ", validLayers[i].shortName(gs));
```

Comment:
What do you think about only putting this comma here if there are 3 or more elements, so that the default case of `library, or application` just prints like

```
`library` or `application`
```

?